airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/NamingHelper.java

```diff
@@ -1,7 +1,33 @@
+/*
```

Comment:
How did it get in initially? Shouldn't have broke the build?

Reply:
it did. see https://github.com/airbytehq/airbyte/commits and https://github.com/airbytehq/airbyte/commit/8c7129c30713a117d8cdc31ed8cc39b75593f524